modules/mono/glue/GodotSharp/GodotSharp/Compat.cs

```diff
@@ -34,6 +34,65 @@ public void AddCodeCompletionOption(CodeCompletionKind type, string displayText,
     }
 }
 
+partial class ConfigFile
+{
+    /// <inheritdoc cref="Save(string)"/>
```

Comment:
```suggestion
    /// <inheritdoc cref="Save(string, bool)"/>
```

I followed the code above, but I think it's wrong, the documentation refers to itself.

Reply:
Yeah, the documentation for `AddCodeCompletionOption` is wrong (fixed in https://github.com/godotengine/godot/pull/80629).